File: Libros2/src/main/java/ec/edu/uta/grupo4/expo/connection/Conexion.java
package ec.edu.uta.grupo4.expo.connection;

import java.sql.Connection;
import java.sql.DriverManager;

/**
 *
 * @author USER
 */
public class Conexion {

    private static final String BASE_URL = "jdbc:mysql://localhost:3306/quintosoa1";
    private static final String USER = "root";
    private static final String PASSWORD = "";

    public static Connection obtenerConexion() {
        try {
            System.out.println("Intentando conectar a: " + BASE_URL);
            Class.forName("com.mysql.cj.jdbc.Driver");
            Connection conn = DriverManager.getConnection(BASE_URL, USER, PASSWORD);
            System.out.println("Conexión establecida correctamente");
            return conn;
        } catch (Exception e) {
            System.out.println("Error de conexión a la base de datos: " + e.getMessage());
            e.printStackTrace();
            return null;
        }
    }
}

File: Libros1/apis/apiLibros/Conexion.php
<?php
  class Conexion {
    public function conectar() {
      $servername = "localhost:3306";
      $dsn = "mysql:host=$servername;dbname=libros";
      $username = "root";
      $password = "";

      try {
        $conn = new PDO($dsn, $username, $password);
      } catch(Exception $e) {
        die("Fallo: " . $e->getMessage());
      }
      return $conn;
    }
  }
?>